tests/acceptance/features/coreApiSharePublicLink2/uploadToPublicLinkShare.feature

```diff
@@ -17,13 +17,14 @@ Feature: upload to a public link share
       | permissions | create   |
       | password    | %public% |
     When the public uploads file "test.txt" with password "%public%" and content "test" using the new public WebDAV API
+    Then the HTTP status code should be "201"
+    And the content of file "/FOLDER/test.txt" for user "Alice" should be "test"
     When the public uploads file "test.txt" with password "%public%" and content "test2" using the new public WebDAV API
```

Comment:
Looks like we have a duplicate scenario here. we can remove this one 
```suggestion
```
and keep the last scenario
https://github.com/owncloud/ocis/blob/8b403a4b86606f365114eb8986d1f887537a0db5/tests/acceptance/features/coreApiSharePublicLink2/uploadToPublicLinkShare.feature#L121-L133